src/core/server/mocks.ts

```diff
@@ -16,9 +16,64 @@
  * specific language governing permissions and limitations
  * under the License.
  */
+import { of } from 'rxjs';
+import { PluginInitializerContext, CoreSetup, CoreStart } from '.';
+import { loggingServiceMock } from './logging/logging_service.mock';
+import { elasticsearchServiceMock } from './elasticsearch/elasticsearch_service.mock';
+import { httpServiceMock } from './http/http_service.mock';
 
 export { configServiceMock } from './config/config_service.mock';
 export { elasticsearchServiceMock } from './elasticsearch/elasticsearch_service.mock';
 export { httpServiceMock } from './http/http_service.mock';
 export { loggingServiceMock } from './logging/logging_service.mock';
 export { SavedObjectsClientMock } from './saved_objects/service/saved_objects_client.mock';
+
+export function pluginInitializerContextConfigMock<T>(defaultValue: T) {
+  return (value?: Partial<T>) => {
```

Comment:
The signature of the function doesn't give a clue what the `value of what`. How about `config`?